stream_alert/alert_processor/outputs/pagerduty.py

```diff
@@ -479,8 +592,38 @@ def dispatch(self, **kwargs):
         incidents_url = self._get_endpoint(self._base_url, self.INCIDENTS_ENDPOINT)
 
         try:
-            success = self._post_request_retry(incidents_url, incident, self._headers, True)
+            incident = self._post_request_retry(incidents_url, incident_data, self._headers, True)
         except OutputRequestFailure:
-            success = False
+            incident = False
 
-        return self._log_status(success)
+        if not incident:
+            LOGGER.error('Could not create main incident, %s', self.__service__)
+            return self._log_status(False)
+
+        # Extract the incident id from the incident that was just created
+        incident_id = incident.get('incident').get('id')
```

Comment:
If the first get returns `None` the second `get` is going to cause an `AttributeError` here. Suggestion:

`incident_id = incident.get('incident', {}).get('id')`

or omit the first get altogether if you're sure it will exist (`incident_id = incident['incident'].get('id')`)